python-checks/src/test/resources/checks/tests/skippedTestNoReason.py

```diff
@@ -0,0 +1,72 @@
+import unittest
+import pytest
+
+
+class MyTest(unittest.TestCase):
+
+    @unittest.skip  # Noncompliant {{Provide a reason for skipping this test.}}
+#   ^^^^^^^^^^^^^^
+    def test_unittest_skip_no_reason(self):
+        self.assertEqual(1 / 0, 99)
+
+    @unittest.skip(test)
+    def test_unittest_skip_no_reason(self):
+        self.assertEqual(1 / 0, 99)
+
+    @unittest.skip("")  # Noncompliant {{Provide a reason for skipping this test.}}
+#                  ^^
+    def test_unittest_skip_empty_reason(self):
+        self.assertEqual(1 / 0, 99)
+
+    @unittest.skip(" ")
+    def test_unittest_skip_blank_reason(self):
+        self.assertEqual(1 / 0, 99)
+
+    @bob.bob
+    def test_qualified_expression_null_symbol(self):
+        self.assertEqual(1 / 0, 99)
+
+# Pytest
+class UsingPytest():
+    @pytest.mark.skip  # Noncompliant {{Provide a reason for skipping this test.}}
+#   ^^^^^^^^^^^^^^^^^
+    def test_pytest_mark_skip_no_reason():
+        assert 1 == 2
+
+    @skip()
+    def test_skip_symbol_not_defined_ok():
+        assert 1 == 2
+
+    @"my string"
+    def test_decorator_without_symbol_ok():
+        assert 1 == 2
+
+    @pytest.mark.skip()  # Noncompliant {{Provide a reason for skipping this test.}}
+#   ^^^^^^^^^^^^^^^^^^^
+    def test_skip_without_args_ko():
+        assert 1 == 2
+
+    @pytest.mark.skip("")  # Noncompliant {{Provide a reason for skipping this test.}}
```

Comment:
For the sake of completeness, I think we could have a test for which the parameter is a string literal (something like `@pytest.mark.skip("a valid reason")`) as we currently don't have one.

Reply:
Correct, will add this !